edsc/collections-collapsed/src/js/edsc-collections-collapsed.coffee

```diff
@@ -106,8 +109,11 @@ class CollectionsCollapsedModel extends KnockoutComponentModel
       domData.clear(flyout)
       dom.remove(flyout)
 
-  hideFlyout: (context, e) =>
-    target = @_getFlyoutTarget(e)
+  _hideFlyouts: (e) => @_destroyAndRemoveFlyout(target) for target in document.getElementsByClassName('flyout-visible')
```

Comment:
This should just hide all if the component's own flyouts, which are maintained in a property.